resources/tests/org.eclipse.core.tests.resources/src/org/eclipse/core/tests/internal/resources/WorkspacePreferencesTest.java

```diff
@@ -243,48 +266,64 @@ public void testSetDescription() throws CoreException {
+		assertThat(WorkspacePreferences.convertStringArraytoString(description.getBuildOrder())).as("check build order")
+				.isEqualTo(expectedPreferences.getString(ResourcesPlugin.PREF_BUILD_ORDER));
+		assertThat(description.isApplyFileStatePolicy()).as("check apply file state policy")
+				.isEqualTo(expectedPreferences.getBoolean(ResourcesPlugin.PREF_APPLY_FILE_STATE_POLICY));
+		assertThat(description.getFileStateLongevity()).as("check file state longevity")
+				.isEqualTo(expectedPreferences.getLong(ResourcesPlugin.PREF_FILE_STATE_LONGEVITY));
+		assertThat(description.getMaxFileStates()).as("check max files states")
+				.isEqualTo(expectedPreferences.getInt(ResourcesPlugin.PREF_MAX_FILE_STATES));
+		assertThat(description.getMaxFileStateSize()).as("check max file state size")
+				.isEqualTo(expectedPreferences.getLong(ResourcesPlugin.PREF_MAX_FILE_STATE_SIZE));
+		assertThat(description.getSnapshotInterval()).as("check snapshot interval")
+				.isEqualTo(expectedPreferences.getLong(ResourcesPlugin.PREF_SNAPSHOT_INTERVAL));
+		assertThat(description.getMaxBuildIterations()).as("check max build iterations")
+				.isEqualTo(expectedPreferences.getLong(ResourcesPlugin.PREF_MAX_BUILD_ITERATIONS));
+		assertThat(description.isKeepDerivedState()).as("check keep derived state")
+				.isEqualTo(expectedPreferences.getBoolean(ResourcesPlugin.PREF_KEEP_DERIVED_STATE));
 	}
 
 	/**
 	 * Compares two workspace description objects..
 	 */
-	public void assertEquals(String message, IWorkspaceDescription description1, IWorkspaceDescription description2) throws ComparisonFailure {
-		assertEquals(message + " - 1", description1.isAutoBuilding(), description2.isAutoBuilding());
-		assertThat(description1.getBuildOrder()).as(message + " - 2").isEqualTo(description2.getBuildOrder());
-		assertEquals(message + " - 3", WorkspacePreferences.convertStringArraytoString(description1.getBuildOrder()), WorkspacePreferences.convertStringArraytoString(description2.getBuildOrder()));
-		assertEquals(message + " - 4", description1.isApplyFileStatePolicy(), description2.isApplyFileStatePolicy());
-		assertEquals(message + " - 5", description1.getFileStateLongevity(), description2.getFileStateLongevity());
-		assertEquals(message + " - 6", description1.getMaxFileStates(), description2.getMaxFileStates());
-		assertEquals(message + " - 7", description1.getMaxFileStateSize(), description2.getMaxFileStateSize());
-		assertEquals(message + " - 8", description1.getSnapshotInterval(), description2.getSnapshotInterval());
-		assertEquals(message + " - 9", description1.getMaxBuildIterations(), description2.getMaxBuildIterations());
-		assertEquals(message + " -10", description1.isKeepDerivedState(), description2.isKeepDerivedState());
-	}
-
-	public static Test suite() {
-		return new WorkspaceSessionTestSuite(PI_RESOURCES_TESTS, WorkspacePreferencesTest.class);
+	public void assertEquals(IWorkspaceDescription actualDescription, IWorkspaceDescription expectedDescription)
```

Comment:
Dito: private method, flip parameters (possibly rename)

```suggestion
	private void assertEquals(IWorkspaceDescription expectedDescription, IWorkspaceDescription actualDescription)
```

Reply:
Fixed with 63513a7.